src/metabase/driver/h2.clj

```diff
@@ -49,7 +49,6 @@
                               :actions/custom            true
                               :datetime-diff             true
                               :now                       true
-                              :csv-uploads               true
```

Comment:
I removed this for now since it's not needed for tests. will reintroduce when H2 is actually supported